component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAPIHandler.java

```diff
@@ -51,122 +47,90 @@ public class BackupCodeAPIHandler {
      * @return List of backup code of the user.
```

Comment:
Lets update the comments to match with method return value

Reply:
addressed with [https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/3426661b07018f6570da38cf3f2431989adbc5df](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/3426661b07018f6570da38cf3f2431989adbc5df)